src/database/models/users.js

```diff
@@ -2,12 +2,29 @@ module.exports = (sequelize, DataTypes) => {
   const Users = sequelize.define(
     'Users',
     {
-      first_name: DataTypes.STRING,
-      last_name: DataTypes.STRING,
-      email: DataTypes.STRING,
-      password: DataTypes.STRING,
-      role: DataTypes.STRING,
-      gender: DataTypes.STRING
+      firstName: {
+        allowNull: false,
+        type: DataTypes.STRING
+      },
+      lastName: {
+        allowNull: false,
+        type: DataTypes.STRING
+      },
+      email: {
+        allowNull: false,
+        unique: true,
+        type: DataTypes.STRING
+      },
+      password: {
+        allowNull: false,
+        type: DataTypes.STRING
+      },
+      role: {
+        type: DataTypes.ENUM('super-admin', 'manager', 'user'),
```

Comment:
Let the roles be
- owner
- admin
- staff

Reply:
Okay, thanks